src/components/configure/keymap/Keymap.tsx

```diff
@@ -169,6 +171,21 @@ export default class Keymap extends React.Component<
     }
   }
 
+  componentDidUpdate(prevProps: KeymapPropsType, prevState: OwnKeymapStateType) {
+    if (prevProps.testMatrix !== this.props.testMatrix) {
+      clearInterval(this.state.interval);
+      if (this.props.testMatrix) {
+        this.setState({ interval: setInterval(() => {
+            this.props.fetchSwitchMatrixState!();
+          }, 100) });
```

Comment:
@rtitmuss When testing this code on my Linux, it worked normally and the CPU load was not almost increased. What OS were you using during developing? We need to test this feature on major OSs, I think.

Reply:
I am developing in Chrome on OS X

Reply:
Ok, I will test this feature on my Windows 10 (Chrome and Edge) and will report the result.